Model/AffirmHeadlessCheckoutManager.php

```diff
@@ -44,7 +46,8 @@ public function __construct(
             $moduleResource,
             $objectManager,
             $helper,
-            $affirmConfig
+            $affirmConfig,
+            $logger
```

Comment:
looks like this parameter was added in the version 3.0.12 but we require 3.0.
Could you also update the composer.json requires at least this version ?